src/components/ManifestForm.js

```diff
@@ -80,16 +67,19 @@ export class ManifestForm extends Component {
   render() {
     const { formValue } = this.state;
     const {
+      addResourcesOpen,
       classes,
       onCancel,
       t,
     } = this.props;
+    if (!addResourcesOpen) return null;
+
     return (
       <form onSubmit={this.formSubmit}>
         <Grid container spacing={2}>
           <Grid item xs={12} sm={8} md={9}>
             <TextField
-              inputRef={this.inputRef}
+              autoFocus
```

Comment:
The changes in this file do not look like they're obviously related to a dev dep bump. Can you say more about these?

Reply:
IIRC, it was motivated by one of the dependency updates breaking a test for this behavior, but MUI's  `autoFocus` provides the behavior we want so we can get out of the whole business by using the upstream code.